klog.go

```diff
@@ -1566,12 +1610,24 @@ func Warningln(args ...interface{}) {
 	logging.println(warningLog, logging.logr, logging.filter, args...)
 }
 
+// WarninglnDepth acts as Warningln but uses depth to determine which call frame to log.
+// WarninglnDepth(0, "msg") is the same as Warningln("msg").
+func WarninglnDepth(depth int, args ...interface{}) {
+	logging.printDepth(warningLog, logging.logr, logging.filter, depth, args...)
```

Comment:
Also should be `logging.printlnDepth` here.

Reply:
There is also FatallnDepth( with similar issue